test/references/CKAN District Spending.txt

```diff
@@ -0,0 +1,21 @@
+register(DataDep(
+    "UK Adur District Council Spending Data",
+    """
+	Dataset: UK: Adur District Council Spending Data
+	Website: https://demo.ckan.org/dataset/adur_district_spending
+	Author: Lucy Chambers,  (Maintainer)
+	License: Creative Commons Attribution Share-Alike
+	Date: July 18, 2017
+	
+	From Spikes Cavell, Spotlight on Spend.\\r
```

Comment:
Hmmm, it is not great that `\\r` is showing up in the output.
That is likely the result of the call to `escape_multiline_string`,
as that is a doubly escaped carriage return.
It is odd that it is showing up, I'ld never noticed it before.

I don;t think needs to be fixed in this PR.
Might be good to open an issue to track it through